.github/workflows/ci-it.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         with:
           submodules: true
       - name: Check License Header
-        uses: apache/skywalking-eyes@9bd5feb86b5817aa6072b008f9866a2c3bbc8587
+        uses: apache/skywalking-eyes@main
```

Comment:
Don't use main to avoid any break possibility. You could update the commit id, which is better and safer.